projects/angular/src/data/tree-view/_tree-view.clarity.scss

```diff
@@ -20,6 +21,73 @@
     //Display
     display: flex;
     align-items: center;
+
+    @include mixins.css-var(
+      font-size,
+      clr-tree-node-font-size,
+      tree-view-variables.$clr-tree-node-font-size,
+      variables.$clr-use-custom-properties
+    );
+    @include mixins.css-var(
+      font-weight,
+      clr-tree-node-font-weight,
+      tree-view-variables.$clr-tree-node-font-weight,
+      variables.$clr-use-custom-properties
+    );
+    @include mixins.css-var(
+      line-height,
+      clr-tree-node-line-height,
+      tree-view-variables.$clr-tree-node-line-height,
+      variables.$clr-use-custom-properties
+    );
+    @include mixins.css-var(
+      letter-spacing,
+      clr-tree-node-letter-spacing,
+      tree-view-variables.$clr-tree-node-letter-spacing,
+      variables.$clr-use-custom-properties
+    );
+
+    // ATM: text and background color changes for hover and click
```

Comment:
Though marked ATM this is not clearly labeled as intentionally commented.

Reply:
Removed in commit f734e6d.

Moved to braking changes branch: [treeview-tm-breaking-changes_cde-940](https://github.com/vmware-clarity/ng-clarity/tree/treeview-tm-breaking-changes_cde-940)